tests/src/test/scala/com/nvidia/spark/rapids/OrcQuerySuite.scala

```diff
@@ -92,4 +97,53 @@ class OrcQuerySuite extends SparkQueryCompareTestSuite {
   ) {
     frame => frame
   }
+
+  private def getColumnEncoding(orcDir: File): (String, String) = {
+    val orcFile = orcDir.listFiles(f => f.getName.endsWith(".orc"))(0)
+    val p = new Path(orcFile.getCanonicalPath)
+    val conf = OrcFile.readerOptions(new Configuration())
+    var reader: Reader = null
+    try {
+      reader = OrcFile.createReader(p, conf)
+      withResource(reader.rows().asInstanceOf[RecordReaderImpl]) { rows =>
+        val stripe: StripeInformation = reader.getStripes.get(0)
+        val stripeFooter = rows.readStripeFooter(stripe)
+        (stripeFooter.getColumns(1).getKind.toString,
```

Comment:
Could we make this more generic?  Like could we get the column encodings for all of the footers and return them as an array? Or would it be better to pass in the columns that we want.  What if there is more than one stripe.  Should we turn this into an assert where we pass in a check for given columns so that we can verify that it is correct for all of the stripes?  If not can we have some comments about why we are only looking at one stripe and have hard coded column numbers?

Reply:
Added comments.